scripts/ghidra/PatchestryDecompileFunctions.java

```diff
@@ -48,13 +55,19 @@
 import java.nio.file.Path;
 
 import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Collections;
 import java.util.Iterator;
 import java.util.Set;
 import java.util.TreeSet;
 
 public class PatchestryDecompileFunctions extends GhidraScript {
+
+    static final int decompilation_timeout = 30;
+
+    private static HashMap<String, DataType> type_map = new HashMap<>();
```

Comment:
Shouldn't be `static` I don't think. Each `single` mode run may lift in different sets of types.